CCDB/src/CcdbApi.cxx

```diff
@@ -897,6 +897,9 @@ void* CcdbApi::navigateURLsAndRetrieveContent(CURL* curl_handle, std::string con
   if (url.find("alien:/", 0) != std::string::npos) {
     return downloadAlienContent(url, tinfo);
   }
+  if (url.find("file:///", 0) != std::string::npos) {
+    return (void*)TFile::Open(url.c_str());
```

Comment:
probably this is right, but there is now a divergence between alien and file protocols: The `alien:` code path is doing sanity type check etc.

I would suggest to modify the existing function to work for `file:` protocolls as well. (The only difference would be that we don't need to check for the GridToken in `downloadAlienContent`.